Build/images/samples/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN dotnet tool install --global PowerShell --version 7.2.4
 # Binder.
 RUN pip install jupytext
 # Make the C# and PowerShell kernels available to Jupytext as well.
-RUN dotnet tool install -g --add-source "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" Microsoft.dotnet-interactive && \
+RUN dotnet tool install -g --add-source "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" Microsoft.dotnet-interactive --version 1.0.355307 && \
```

Comment:
Do we know why it doesn't just grab the latest version by default? Do we have to specify it here?

Reply:
The command does grab the latest version by default, but the versions after this one have dropped the support of .NET 6.0. 
See [Versions](https://www.nuget.org/packages/Microsoft.dotnet-interactive/1.0.355307#versions-body-tab) / {Click a specific version} / {see the [Frameworks](https://www.nuget.org/packages/Microsoft.dotnet-interactive/1.0.355307#supportedframeworks-body-tab) tab for that version}.

I can add this explanation as the code comments. Shall I add?

Reply:
Yah having a comment explaining the reason for the specific version would be good so that devs in the future looking at this know why that version specifically and don't updated it to the latest.

Reply:
Done.